src/Storage/TSStorageManager.h

```diff
@@ -21,9 +17,15 @@ extern NSString *const TSUIDatabaseConnectionDidUpdateNotification;
 @interface TSStorageManager : NSObject
 
 + (instancetype)sharedManager;
+
+/**
+ * Database password might not be accessible if the user just restarted the device and hasn't unlocked. If that's the
+ * case, we can't usefully start the app until they unlock.
```

Comment:
I think this comment could be a little misleading. Perhaps:

```
* Returns NO if:
*
* - Keychain is locked because device has just been restarted.
* - Password could not be retrieved because of a keychain error.
```